### PR TITLE
Iframe buttons

### PR DESCRIPTION
**Pull request by keur**

### CODE REVIEW

File: public/html/decrypt.html [hunk 1]
```diff
@@ -71,15 +71,36 @@
...
 
 <body>
   <aside>
     <section>
       <div>
-        <textarea id="decrypt-box" lang="en" dir="ltr" spellcheck="false"></textarea>
+        <textarea id="decrypt-box" readonly lang="en" dir="ltr" spellcheck="false"></textarea>
       </div>
+      <div>
+        <button id="copy" disabled type="button">Copy</button>
```
Comment (yvbbrjdr) [inline public/html/decrypt.html]: Do we want to use bootstrap?

File: src/background/background.ts [hunk 2]
```diff
@@ -25,8 +25,26 @@ chrome.runtime.onMessage.addListener(
...
+          (response: any): void => {
+            sendResponse(response);
+          }
+        );
+        break;
+      case 'proxyPickerSelect':
+        sendMessageActiveTab(
+          { request: 'pickerSelect', data: null },
+          (response: any): void => {
+            console.log(response);
```
Comment (yvbbrjdr) [inline src/background/background.ts]: ?

File: src/background/background.ts [hunk 3]
```diff
@@ -25,8 +25,26 @@ chrome.runtime.onMessage.addListener(
...
+            sendResponse(response);
+          }
+        );
+        break;
       default:
         break;
     }
+    return true;
```
Comment (yvbbrjdr) [inline src/background/background.ts]: What's the point of the return value if it never returns false?

File: src/content/inject.ts [hunk 4]
```diff
@@ -24,11 +24,24 @@ window.onload = (): void => {
         }
       );
     },
+    (_: EventTarget) => {
+      chrome.runtime.sendMessage(
```
Comment (yvbbrjdr) [inline src/content/inject.ts]: If the user clicks on something that's not decryptable, the page is effectively unusable.

File: src/background/background.ts (hunk 2 above)
Comment (keur) [reply src/background/background.ts]: This is an outdated commit.

File: public/html/decrypt.html (hunk 1 above)
Comment (keur) [reply public/html/decrypt.html]: No, I don't want to pull in bootstrap for the element picker. I'm not opposed, but we can do another PR for it later.

File: src/content/inject.ts (hunk 4 above)
Comment (keur) [reply src/content/inject.ts]: Thanks for catching this!

**Comment by keur** [comment]
@yvbbrjdr Oops. I didn't notice this issue on Chromium because it has a much lower recursion depth. Had a logic bug with `deactivate` calling itself. Fixed and force pushed.

**Pull request merged**